packages/rocketchat-statistics/server/functions/get.js

```diff
@@ -110,6 +110,10 @@ RocketChat.statistics.get = function _getStatistics() {
 		platform: process.env.DEPLOY_PLATFORM || 'selfinstall',
 	};
 
+	statistics.uploadsTotal = RocketChat.models.Uploads.find().count();
+	const [result] = Promise.await(RocketChat.models.Uploads.model.rawCollection().aggregate([{ $group: { _id: 'total', total: { $sum: '$size' } } }]).toArray());
+	statistics.uploadsTotalSize = result.total;
```

Comment:
```suggestion
	statistics.uploadsTotalSize = result ? result.total : 0;
```
It will break if uploads collection is empty